### PR TITLE
pradhuman09.json

### PR DESCRIPTION
**Pull request by pradhuman09**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6439"></a>

### CODE REVIEW

File: data/pradhuman09.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "type" : "personal",
+  "bio":   "this is my personal project"
+}
+
+"Links":[
+  {
+    "name": "Follow me on Github",
+    "url": "https://github.com/pradhuman09",
+    "icon": "FagitHub"
+  },
```
Comment (SaraJaoude) [inline data/pradhuman09.json:12]:
```suggestion
	"bio": "this is my personal project",

	"links": [{
			"name": "Follow me on Github",
			"url": "https://github.com/pradhuman09",
			"icon": "FagitHub"
		}
	]
}
```

File: data/pradhuman09.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+	"links": [{
+			"name": "Follow me on Github",
+			"url": "https://github.com/pradhuman09",
+			"icon": "FagitHub"
+		}
+	]
+}
+  
+]
```
Comment (SaraJaoude) [inline data/pradhuman09.json:14]:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Review by Aniket1026 — COMMENTED** [review]
Sorry for the confusion changes are fine.
As i cannot delete the comment i'm updating

**Pull request merged**